app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir=. --presets=es2015",
```

Comment:
In most of my projects I start a build watcher to continuously build my ES2015 code. My AVA tests exercise the build output, I don't use `babel-register`. I find it a bit weird to load the code-to-be-tested from a `dist` folder.

Also, while you can use `main` in the `package.json` to point at `dist/index.js`, that doesn't work if you make other modules part of your interface. E.g. with AVA that'd be `ava/api`. Though you could write wrapper modules and commit those.

Finally I tend to put the non-public modules in a `lib` folder, so at the end of the day it's not too much hassle to manage the individual files. Though it's more work than using a `dist` folder.